### PR TITLE
gateway: import solarnet ansible scripts

### PR DESCRIPTION
**Pull request by lgierth**
Imports the current master HEAD:
https://github.com/protocol/infrastructure/commit/35c5e1f5af908b703f5ce5b851ea484681a6637b

License: MIT
Signed-off-by: Lars Gierth <larsg@systemli.org>

### CODE REVIEW

**Comment by lgierth** [comment]
The export on the other end: https://github.com/protocol/infrastructure/pull/71

**Comment by jbenet** [comment]
LGTM! :+1:

File: solarnet/authorized_keys [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAvs4R6b65em07jLEgQwkz0/jLPx/VRH/wjkkLz/diojnJ0ridZ6ixVYcwEJG97XJJAa/T/kRaVxGryolHpXHREgEirKtV9D1TYLAMhr+L80bKmVH6b/btBJKccw5NrN1vo3qSayR+OLkLLHn2bHwEriR0zk0/FeCvN1Kiei9GQIggJ/jchObB5T6kqdQGs/fHRyk22N4UtrqzYfivF9I7Dp+DdiSo/rBDOVW1bdlSsiAXB0gzcv6FKemtxGFg1ND6m7syv2gplEI1CyNUzT6rfonbUi5cj/ltdgEtqzq/JJxpN8biO7twNo9KARE5moNSjR6Kykz15P7sLxHf0y+tDQ== jbenet
```
Comment (jbenet) [inline solarnet/authorized_keys]: these are just rsa-keys but maybe not needed.

File: solarnet/playbooks/domains.yaml [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+      do:
+          DO_API_KEY: "{{ lookup('env','DIGITAL_OCEAN_API_KEY') }}"
+          DO_CLIENT_ID: "{{  lookup('env','DIGITAL_OCEAN_CLIENT_ID') }}"
+  tasks:
+    - name: "install dopy"
+      pip: name=dopy
+    - environment: do
+      digital_ocean_domain:
+        state: present
+        name: uranus.nyc.srv.protocol-dev.com
```
Comment (jbenet) [inline solarnet/playbooks/domains.yaml]: we probably want to move off of `protocol-dev.com` for this and to `i.ipfs.io` -- do we need the full `"<host>.<city>.srv."` prefix? (not sure where it came from originally).

Comment (jbenet) [inline solarnet/playbooks/domains.yaml]: we should be able to manage the `i.ipfs.io` domain from DO. i'll make sure you have creds.

Comment (whyrusleeping) [inline solarnet/playbooks/domains.yaml]: i like <nodename>.i.ipfs.io

Comment (lgierth) [inline solarnet/playbooks/domains.yaml]: Yeah no idea, so far I've left domains.yml out of the cleanup. I'm cool with nodename.i.ipfs.io. There isn't really a good reason to partition the domain by city or datacenter, at the current scale of things.

File: solarnet/authorized_keys (hunk 1 above)
Comment (lgierth) [inline solarnet/authorized_keys]: Yep gonna move to secrets.yml

**Pull request merged**